src/Microsoft.AspNetCore.Server.HttpSys/HttpSysOptions.cs

```diff
@@ -55,6 +57,29 @@ public HttpSysOptions()
         public bool ThrowWriteExceptions { get; set; }
 
         /// <summary>
+        /// Gets or sets the maximum number of concurrent connections to accept, -1 for infinite, or null to
+        /// use the machine wide setting from the registry. The default value is null.
+        /// </summary>
+        public long? MaxConnections
```

Comment:
Would be good to add tests asserting the default value = null, that users can set valid values, and the out of range exception.